hack/realpath.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+import os.path
+import optparse
+
+
+def realpath(files):
+    for file_ in files:
+        print(os.path.realpath(file_))
+
+
+if __name__ == '__main__':
+    parser = optparse.OptionParser()
+    (_, pos_args) = parser.parse_args()
```

Comment:
Not sure why you need `optparse`, could as well use `sys.argv[1:]` or something alike. Anyway, nothing major.

Reply:
If you call `python ./hack/realpath.py -s /path/to/file` you have a message that option -s doesn't exist instead of some "realpath" of "-s"

Reply:
Sure, yet at the same time:
- We don't use `-s` in `Makefile`
- Now this tool simply disregards options and as such behaves different than `realpath` anyway, actually providing an incorrect result when running with e.g. `-s` :)

Reply:
Ah, didn't consider `parse_args` raises when unknown args are found.

All good then :+1: